sustainability_account_accountant/__manifest__.py

```diff
@@ -10,7 +10,7 @@
     "maintainers": ["jguenat", "bonnetadam", "jacopobacci"],
     "development_status": "Production/Stable",
     "website": "https://www.open-net.ch",
-    "depends": ["sustainability", "account_accountant"],
```

Comment:
Is there a reason to remove this modules from the depends ?

Reply:
No. I was just trying to get to the part where the tests actually start. You can definitely wipe my commit away.